src/views/TheWalletView.vue

```diff
@@ -78,7 +78,12 @@ export default {
     this.web3.eth.clearSubscriptions();
   },
   methods: {
-    ...mapActions('wallet', ['setBlockNumber', 'setTokens', 'setWallet']),
+    ...mapActions('wallet', [
+      'setBlockNumber',
+      'setTokens',
+      'setWallet',
+      'removeWallet'
```

Comment:
doesn't seem like this is being used.

Reply:
Oh yeah, thanks.  The idea was if metamask was sending an empty account array every time the user locked metamask, we could use that to just log them out.  However, looks like the empty account / accountsChanged event is only sent on first time lock

I'll remove this change.